notebooks/01_nexus_setup.ipynb

```diff
@@ -0,0 +1,133 @@
+     ]
+    }
+   ],
+   "source": [
+    "TOKEN = getpass.getpass()"
```

Comment:
Coul bluepyentity's get_token be used here?

Reply:
Yes, very likely. I was considering trying this but decided that I'd just bring these over and check they still work. 

I'll do any changes/improvements in another PR. Created an issue of this: #20

Reply:
Sure, fair enough